### ISSUE TITLE
Service Key Policies

### ISSUE THREAD

**Issue by agregory999** [pivotal-cf/docs-aws-services#10]
For the section in the docs called "Service Key Policies", we don't understand the relationship for IAM users and the policies.  For example, we created a user called pcf-broker-iam and attached the "PCFInstallationPolicy" to it.  But the doc doesn't ask us to attach the other 4 policies (for S3, SQS, dynamo ) , so what role or user are they part of? Please clarify in the docs.  Or maybe include a screen shot or table summary of what the policies and users should look like.

**Comment by bentarnoff** [comment]
Hey @xtreme-alvin-hsieh could you speak to this?

**Comment by jncd** [comment]
Hi @xtreme-alvin-hsieh It would be nice to fix this issue... You can pair with @Pivotal-Christopher-Wong !

**Comment by xtreme-alvin-hsieh** [comment]
Hi @agregory999. 

When you "created a user called pcf-broker-iam and attached the "PCFInstallationPolicy" This step is required and correct. You can think of this user as the "root" user that has all the permissions specified in "PCFInstallationPolicy" which has multiple services. The creation of all the services created through the cf create-service command is done through this user.

The confusion is what are the service key policies for. The docs states that AWS SB can create service keys and, "Creating a service key creates an IAM user with a templated policy, and provides the App Developer with Access Key credentials to the actions listed in the policies below." What this means is that say you've already created a service (through the root user mentioned above), ie s3, and you want to give another developer access to it, the broker will create a new user, with the permissions specified in SERVICE key policy, unique to that service.

In the docs, you'll see that we create a service key policy for each service, and the name of each policy is input to each service section (tab) of the tile. 

To summarize, service key policy is service specific, it specifies the permissions that a newly created user would have for that type of service. Since this user is created to allow access to a particular instance of the service, the user will have the specified permission only to the instance that they were created for access.
